src/Admin/AbstractAdmin.php

```diff
@@ -812,7 +814,14 @@ public function getFilterParameters()
 
         // build the values array
         if ($this->hasRequest()) {
-            $filters = $this->request->query->get('filter', []);
+            /** @var InputBag|ParameterBag $bag */
```

Comment:
Is this needed? on the other class you didnt add it, does phpstan complain?

Reply:
psalm complains without it

Reply:
```
ERROR: NullReference - src/Admin/AbstractAdmin.php:822:34 - Cannot call method get on null value (see https://psalm.dev/016)
                $filters = $bag->get('filter', []);


------------------------------
1 errors found
------------------------------